inc/features/blocks/personalization/components/variant-toolbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const { IconButton } = wp.components;
+const { __ } = wp.i18n;
+
+const VariantToolbar = props => {
+	const {
+		canRemove,
+		isFallback,
+		onCopy,
+		onRemove,
+	} = props;
+
+	return (
+		<div className="altis-experience-block-header__toolbar">
+			<IconButton
+				icon='migrate'
```

Comment:
Static props always want double-quoting

```suggestion
				icon="migrate"
```